lib/iron_bank/payment_run.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module IronBank
+  # Create payment run
+  # https://www.zuora.com/developer/api-reference/#operation/POST_PaymentRun
+  #
+  class PaymentRun
+    ENDPOINT = "/v1/payment-runs"
+
+    def self.create(params)
+      payload = IronBank::Object.new(params).deep_camelize(type: :lower)
+      body    = IronBank.client.connection.post(ENDPOINT, payload).body
+      success = body.fetch("success", false)
```

Comment:
I know Zuora has a few different ways of telling us if it was "success", is this sufficient in this case?

Reply:
If there is an error code we identified, the `raise_error` middleware will take care of raising the adequate exception. This is a later verification, similar to what we do in `IronBank::Actions::Create`

Reply:
Ah true, I forgot about implementing that awesome middleware.